src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -48,6 +57,8 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
   {
     def vantage = project.extensions.getByType(VantageExtension)
 
+    project.configurations.removeAll{it.name in doNotResolveConfigurations}
```

Comment:
you should be able to filter them out in the streams below, e.g. with `findAll`.  This code is actually going to mutate the actual gradle configuration.